e2e/tests/block/k8s/integration/k8sBlockStorageIntegration_test.go

```diff
@@ -0,0 +1,159 @@
+package integration
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/rook/rook/e2e/framework/clients"
+	"github.com/rook/rook/e2e/framework/contracts"
+	"github.com/rook/rook/e2e/framework/enums"
+	"github.com/rook/rook/e2e/framework/manager"
+	"github.com/rook/rook/e2e/framework/objects"
+	"github.com/rook/rook/e2e/framework/utils"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"html/template"
+	"testing"
+)
+
+// Rook Block Storage integration test
+// Start MySql database that is using rook provisoned block storage.
+// Make sure database is functional
+var (
+	env      objects.EnvironmentManifest
+	platform enums.RookPlatformType
+)
+
+//One init function per package - initializes Rook infra and installs rook(if needed based on flags)
+func init() {
+	env = objects.NewManifest()
+	var err error
+
+	rookPlatform, err := enums.GetRookPlatFormTypeFromString(env.Platform)
+	if err != nil {
+		panic(fmt.Errorf("Cannot get platform", err))
+	}
+
+	k8sVersion, _ := enums.GetK8sVersionFromString(env.K8sVersion)
+
+	rookTag := env.RookTag
+	if err != nil {
+		panic(fmt.Errorf("Rook Tag is required", err))
+	}
+
+	err, rookInfra := rook_test_infra.GetRookTestInfraManager(rookPlatform, true, k8sVersion)
+	if err != nil {
+		panic(fmt.Errorf("Error during Rook Infra Setup", err))
+	}
+	skipRookInstall := env.SkipInstallRook == "true"
+
+	rookInfra.ValidateAndSetupTestPlatform(skipRookInstall)
+
+	err = rookInfra.InstallRook(rookTag, skipRookInstall)
+	if err != nil {
+		panic(fmt.Errorf("Error during Rook Infra Setup", err))
+	}
+	platform = rookInfra.GetRookPlatform()
+}
+
+func TestK8sBlockIntegration(t *testing.T) {
+	suite.Run(t, new(K8sBlockEnd2EndIntegrationSuite))
+}
+
+type K8sBlockEnd2EndIntegrationSuite struct {
+	suite.Suite
+	testClient       *clients.TestClient
+	bc               contracts.BlockOperator
+	kh               *utils.K8sHelper
+	init_blockCount  int
+	storageclassPath string
+	mysqlappPath     string
+	db               *utils.MySqlHelper
+}
+
+//Test set up - does the following in order
+//create pool and storage class, create a PVC, Create a MySQL app/service that uses pvc
+func (s *K8sBlockEnd2EndIntegrationSuite) SetupSuite() {
+
+	var err error
+
+	s.testClient, err = clients.CreateTestClient(platform)
+	require.Nil(s.T(), err)
+
+	s.bc = s.testClient.GetBlockClient()
+	s.kh = utils.CreatK8sHelper()
+	initialBlocks, err := s.bc.BlockList()
+	require.Nil(s.T(), err)
+	s.init_blockCount = len(initialBlocks)
+	s.storageclassPath = "../../../../data/block/storageclass_pool.tmpl"
```

Comment:
I was using GOPATH where the functions was being called by other packages(messing up relative path). But I should probably use it make tests consistent.